android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java

```diff
@@ -290,49 +291,73 @@ private void setMACFiltering(MethodCall poCall, Result poResult) {
      * (e.g., {@code 01a243f405}).
      */
     private void getWiFiAPSSID(Result poResult) {
-        android.net.wifi.WifiConfiguration oWiFiConfig = moWiFiAPManager.getWifiApConfiguration();
-        if (oWiFiConfig != null && oWiFiConfig.SSID != null) {
-            poResult.success(oWiFiConfig.SSID);
-            return;
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.Q) {
+            android.net.wifi.WifiConfiguration oWiFiConfig = moWiFiAPManager.getWifiApConfiguration();
+
+            if (oWiFiConfig != null && oWiFiConfig.SSID != null) {
+                poResult.success(oWiFiConfig.SSID);
+                return;
+            }
+
+            poResult.error("Exception", "SSID not found", null);
         }
-        poResult.error("Exception", "SSID not found", null);
+            else {
+                poResult.error("Exception [getWiFiAPSSID]", "Getting SSID name doesn't work on API level >= 29", null);
```

Comment:
Really "nitpicking" but maybe "is not supported" is better than "doesn't work" ? Again: it's nitpicking, but may be more correct (as one can still do it, but not with this APIs).
Same comment applies for other similar messages below.

Reply:
My apologies, I'll change this.